support aws ebs snapshot based backup and restore (#2065)
pingcap/docs-tidb-operator: en/backup-restore-cr.md

    * `full`: back up all databases in a TiDB cluster.
    * `db`: back up a specified database in a TiDB cluster.
    * `table`: back up a specified table in a TiDB cluster.

* `.spec.backupMode`: the backup mode. The default value is `snapshot`, which means snapshot backup in the KV layer. This field is valid only for backup and has three value options currently:
    * `snapshot`: back up snapshots in the KV layer.
    * `volume-snapshot`: back up volume snapshots.
    * `log`: back up log data in real time in the KV layer.

* `.spec.restoreMode`: the restore mode. The default value is `snapshot`, which means snapshot restore in the KV layer. This field is valid only for restore and has three value options currently:
    * `snapshot`: restore snapshots in the KV layer.
    * `volume-snapshot`: restore volume snapshots.

>     * `log`: restore cluster data to a specified time based on snapshots and log data.

Reviewer: @fengou1 就按照你说的来写，这样可以吗？ 
* `log`: after full data restore based on snapshots in the KV layer, you can use this option to restore cluster incremental data to a specified time based on logs.

>     * `log`: restore cluster data to a specified time based on snapshots and log data.

Reviewer 2: 找相关的同事聊了下，这里两个错误
1. pitr 不是 log,  backupMode 里面才是 log,  restoreMode 只能是 pitr
2. restore， 是一次性的，但是会先 restore snapshots， 再 restore log，恢复到某个 时间点。

>     * `log`: restore cluster data to a specified time based on snapshots and log data.

Reviewer 2: ```suggestion
    * `pitr`: restore cluster data to some point in time based on snapshots and log data.
```
